rudder-server-root/SPECS/rudder-server-root.spec

```diff
@@ -52,7 +52,7 @@ Source8: rudder-server-root
 BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 BuildArch: noarch
 
-Requires: rudder-jetty, rudder-webapp, rudder-inventory-endpoint, rudder-inventory-ldap, rudder-reports, rudder-agent, curl
+Requires: rudder-jetty, rudder-webapp, rudder-inventory-endpoint, rudder-inventory-ldap, rudder-reports, Rudder::Agent, curl
```

Comment:
I'm a bit concerned by this, because if it ends up asking the user to make a choice, we're losing out on ease-of-install. On Debian, this would be the case, but I'm not sure on SLES/RHEL.